free_fleet_adapter/tests/integration/test_nav2_robot_adapter.py

```diff
@@ -128,10 +130,11 @@ def test_robot_stop_without_command(self):
             fleet_handle=None,
             tf_buffer=tf_buffer
         )
-        assert robot_adapter.execution is None
+        nav_handle = ExecutionHandle(None)
```

Comment:
```suggestion
        robot_adapter.nav_handle = ExecutionHandle(None)
```
to be used by `robot_adapter.stop(None)`, the other references to `nav_handle` in this test should be changed too